### ISSUE TITLE
Super mode not working 

### ISSUE THREAD

**Issue by avi7271** [EVLF/Cypher-Rat-Source-Code#46]
I have turn on super mod while building apk but when I try to unistall it unistall easily without any error or anything, like in emulator when I try to unistall supr mod apk it minimize the unistall option but in real life it doesn't do anything like that.

**Comment by EVLF** [comment]
what the phone language ?
cuz Anti-uninstall does not support all phones languages

**Comment by avi7271** [comment]
phone language is english, it work for 20minutes, then disable or after
restarting. after 1 or 2 day some function like keyloger, camera, live
screen stop working.

**Comment by EVLF** [comment]
yes some people  reported that , after the phone restart accessibility service stop working
i will fix it next update

**Comment by sachinkuma5** [comment]
Yes i have same problem

**Comment by avi7271** [comment]
Thanx you, and also fix bug which stop app to auto start after phone
restart and cause app to partial work, like only storage access work.
